workshops/personal_website/README.md

```diff
@@ -37,38 +47,47 @@ _Pro tip: if you want a username that's already taken, you can often get it by [
 
 GitHub allows us to host our website using a service called GitHub Pages. This means that we can put the files of our website on GitHub and GitHub will give us a URL that we can share with the world.  
 
-1. Go to https://github.com
-2. Click the green "Create repository" button
-
-   > ![](img/gh_new_repo_button.gif)
+1. Go to https://github.com/new
 
-3. Under "Repository name" write `USERNAME.github.io` (replace `USERNAME` with your actual username). You have to name your repository this in order for GitHub Pages to work correctly (we'll talk more about GitHub Pages later)
+2. Under "Repository name" write `USERNAME.github.io` (replace `USERNAME` with your actual username). You have to name your repository this in order for GitHub Pages to work correctly (we'll talk more about GitHub Pages later)
 
    > ![](img/gh_create_new_repo.png)
 
-5. Yay! You've created your very first repository! Now, copy the "HTTPS link" at the top. You will need this link for the next step.
+3. Yay! You've created your very first repository! Now, copy the "HTTPS link" at the top. You will need this link for the next step.
 
    > ![](img/gh_copy_link.gif)
 
 ### 3) Signing Up for Cloud9
 
-We'll be using a service called Cloud9 to write, save, and organize all of the code we'll write in Hack Club.
+We'll be using a service called Cloud9 to write, save, and organize all the code we'll write in Hack Club.
 
-1. In a new tab, open https://c9.io/
+1. In a new tab, open https://c9.io/.
 2. Enter your email and go through the signup process.
 
    > ![](img/c9_start_signup.gif)
 
-   If Cloud9 asks for a credit card, use the following info. This is an empty
-   prepaid debit card -- it'll be enough for verification, but you won't be able
-   to make any actual charges to it.
+   If Cloud9 asks for a credit card, use the following info. This is an empty prepaid debit card -- it'll be enough for verification, but you won't be able to make any actual charges to it.
 
    > ![](img/c9_credit_card.png)
-   >
-   > | Field       | Value      |
-   > | ----------- | ---------- |
-   > | Full name   | Zach Latta |
-   > | Postal code | 94114      |
+
+   > <table>
+   >   <thead>
+   >     <tr>
+   >       <th>Field</th>
+   >       <th>Value</th>
+   >     </tr>
+   >   </thead>
+   >   <tbody>
+   >     <tr>
+   >       <td>Full name</td>
+   >       <td>Zach Latta</td>
+   >     </tr>
+   >     <tr>
+   >       <td>Postal code</td>
+   >       <td>94114</td>
+   >     </tr>
+   >   </tbody>
+   > </table>
```

Comment:
There's an issue with how we're rendering markdown (it's in markserv and lecture-hall) that doesn't properly render markdown tables inside a list.